plugins/modules/power_system.py

```diff
@@ -110,8 +110,9 @@
             - C(modify_hwres) Makes hardware resource configurations of specified I(system_name).
             - C(enable_pcm) Enables the Performance and Capacity Monitoring for specified types of utilization data.
             - C(disable_pcm) Disables the Performance and Capacity Monitoring for specified types of utilization data.
+            - C(list_pcm) Provides the status of the Performance and Capacity Monitoring of the managed system
         type: str
-        choices: ['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm']
+        choices: ['poweron', 'poweroff', 'modify_syscfg', 'modify_hwres', 'enable_pcm', 'disable_pcm', 'list_pcm']
```

Comment:
this documentation still shows as list_pcm, which should be changed to ppm_facts